frontend/src/Page.js

```diff
@@ -7,26 +7,46 @@ import CreateEmployee from './employees/CreateEmployee';
 import UpdateEmployee from './employees/UpdateEmployee';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
-
+import LogOut from './LogOut';
 
 import {
     BrowserRouter as Router,
     Switch,
     Route,
     Redirect
 } from "react-router-dom";
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Ezt lehet hogy egy globálisabb helyen kellene behúzni, biztos máshova is kelleni fog a bootstrap css

Reply:
törölve innen.